crates/jstzd/tests/octez_client_test.rs

```diff
@@ -351,3 +358,185 @@ async fn send_rollup_inbox_message() {
     let _ = baker.kill().await;
     let _ = octez_node.kill().await;
 }
+
+#[derive(Deserialize)]
+struct OutputProof {
+    pub commitment: String,
+    pub proof: String,
+}
+
+#[tokio::test(flavor = "multi_thread")]
+async fn execute_rollup_outbox_message() {
+    let rollup_address = "sr1Uuiucg1wk5aovEY2dj1ZBsqjwxndrSaao";
+    // this is the destination contract where the outbox messages target
+    // this address is sealed in the rollup code
+    let contract_address = "KT1TFAweS9bMBetdDB3ndFicJWAEMb8MtSrK";
+    let installer_path = Path::new(std::env!("CARGO_MANIFEST_DIR")).join(format!(
+        "tests/resources/rollup/{rollup_address}/installer.hex"
+    ));
+    let preimages_dir = Path::new(std::env!("CARGO_MANIFEST_DIR"))
+        .join(format!("tests/resources/rollup/{rollup_address}/preimages"));
+    let contract_path = Path::new(std::env!("CARGO_MANIFEST_DIR"))
+        .join(format!("tests/resources/contract/{contract_address}.json"));
+    let params = set_up_parameters_for_outbox_message(
+        rollup_address,
+        contract_address,
+        &installer_path,
+        &contract_path,
+    )
+    .await;
+
+    let (mut octez_node, octez_client, mut baker) =
+        setup(Some(params.parameter_file().path().to_path_buf())).await;
+    let mut rollup = spawn_rollup(
```

Comment:
Nit: Would be nice if the builder could take an octez node and client to instantiate it. Not required for this PR.

Reply:
You mean `OctezRollupConfigBuilder`? If so, it was like that at some point but I changed it because it introduced dependencies on the node and the client instances and I ran into some issues because of that. Perhaps we can revisit that later on.

Reply:
Yes the `OctezRollupConfigBuilder`, because that is kind of what `spawn_rollup` is doing.